common/src/main/java/io/netty/util/internal/NativeLibraryLoader.java

```diff
@@ -125,10 +125,15 @@ public static void load(String originalName, ClassLoader loader) {
             }
         }
 
-        if (url == null) {
-            // Fall back to normal loading of JNI stuff
+        try {
+            // first try to load from java.library.path
             loadLibrary(loader, name, false);
+            logger.info("{} was loaded from java.libary.path", name);
```

Comment:
I personally prefer the info level, since it would point to potential out-of-date libraries on disk, if someone messes up the java.library.path, but I will change it to logger.debug if you think debug is better.